hoomd/filter/filter_.py

```diff
@@ -11,4 +14,5 @@ def __str__(self):
 
     def __call__(self, state):
```

Comment:
Remove this `__call__` method and instead export `getSelectedTags` as `__call__` in `export_filters.cc`.

Reply:
Wait, never-mind. I realize now that this needs access to the python only `state`.